cttportfolio.css

```diff
@@ -0,0 +1,982 @@
+	25% {
+		visibility: hidden;
+		opacity: 0;
+	}
+	40% {
+		visibility: visible;
+		opacity: 1;
+	}
+	100% {
+		visibility: visible;
+		opacity: 1;
+	}
+}
+
+@keyframes fadeIn3 {
+	40% {
+		visibility: hidden;
+		opacity: 0;
+	}
+	65% {
+		visibility: visible;
+		opacity: 1;
+	}
+	100% {
+		visibility: visible;
+		opacity: 1;
+	}
+}
+
+
+/*{ Keeping this stored here for now for access later if needed with browser testing
+    -webkit-animation: fadein1 1s; /* Safari, Chrome and Opera > 12.1 
+       -moz-animation: fadein1 1s; /* Firefox < 16 
+        -ms-animation: fadein1 1s; /* Internet Explorer 
+         -o-animation: fadein1 1s; /* Opera < 12.1 
+            animation: fadein 1s;
+}/*
+/*======================================== NAV SECTION ========================================*/
+
+nav {
+	display: flex;
+	padding: 0 5%;
+	justify-content: space-between;
+	align-items: center;
+	position: fixed;
+	top: 0;
+	width: 100vw;
+	z-index: 1;
+	margin: 10px 0;
+}
+
+nav a,
+nav p {
+	text-decoration: none;
+	color: rgb(248, 248, 255);
+	text-transform: uppercase;
+}
+
+.navlogo>img {
+	padding-right: 15px;
+	height: 100%;
+	position: relative;
+	bottom: 4px;
+	box-shadow: none;
+}
+
+.navlogo>p {
+	font-size: 2.5rem;
+	font-weight: 700;
+}
+
+.navlogo {
+	display: flex;
+	align-items: center;
+	height: 80px;
+}
+
+.navlinks>a {
+	font-size: 2rem;
+	margin-left: 30px;
+	padding: 15px 10px;
+	padding-top: 18px;
+	border: none;
+	border-radius: 5px;
+	font-weight: 600;
+}
+
+.navlinks a:hover {
+	border: 1px solid rgb(248, 248, 255);
+}
+
+.nav_dropdown {
+	display: none;
+}
+
+
+/*======================================== HEADER SECTION ========================================*/
+
+header {
+	width: 100vw;
+	height: 100vh;
+	position: relative;
+	z-index: -1;
+}
+
+#frontpagevideo,
+#frontpagevideo[poster] {
+	object-fit: cover;
+	max-width: 100%;
+	position: fixed;
+}
+
+.headertext {
+	position: absolute;
+	top: 0;
+	left: 0;
+	width: inherit;
+	height: inherit;
+	display: flex;
+	flex-direction: column;
+	justify-content: center;
+	padding-left: 20%;
+}
+
+h1 {
+	font-size: 2.5rem;
+	color: white;
+}
+
+h2 {
+	font-size: 1.8rem;
+	text-transform: uppercase;
+	color: rgb(248, 248, 255);
+}
+
+
+/*======================================== ABOUT ME SECTION ========================================*/
+
+.aboutme {
+	height: 1200px;
+	display: flex;
+	flex-direction: column;
+	justify-content: space-between;
+}
+
+.aboutme>* {
+	margin-bottom: 50px;
+	border-radius: 5px;
+}
+
+.aboutme>button {
+	margin: 0 auto;
+}
+
+.amcontainer {
+	height: 500px;
+	display: flex;
+}
+
+.am_main>* {
+	padding: 25px;
+}
+
+.am_img {
+	width: 30%;
+}
+
+.am_img>img {
+	max-width: 100%;
+	max-height: 100%;
+	border-radius: 7px;
+}
+
+
+/*======================================== SKILLS SECTION ========================================*/
+
+.am_sub {
+	margin-top: 50px;
+}
+
+.am_sub_img {
+	position: relative;
+}
+
+#am_sub_ar {
+	opacity: 0;
+}
+
+.am_sub_content {
```

Comment:
not sure about this. I know it works but as far as I can tell, it's just a purple box ?